src/Core/App.php

```diff
@@ -187,7 +187,7 @@ protected static function _classExistsInBase(string $name, string $namespace): b
      */
     public static function path(string $type, ?string $plugin = null): array
     {
-        if ($plugin === null && $type[0] === strtolower($type[0])) {
+        if ($plugin === null) {
             return (array)Configure::read('App.paths.' . $type);
```

Comment:
Both `/** @psalm-suppress PossiblyNullArgument */` can now be removed below because `$plugin` never `null`.

Reply:
Couldn't a null still be provided as an argument?

Reply:
When `$plugin === null`, function exits here, so few lines below (GitHub didn't let comment on them directly) `Plugin::templatePath($plugin)` and `Plugin::path($plugin)` always passes `string`.

Reply:
Yup I'll remove those annotations.